src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, Dispatch, SetStateAction } from 'react';
+import { Divider, List, ListItem, ListItemText, ListItemIcon, Box, CircularProgress, ListSubheader } from '@mui/material';
+import { Link } from 'react-router-dom';
+
+import useStyles from './styles';
+
+import githubWhite from '../assets/logo/github_white.png';
+// import githubblack from '../assets/logo/github_black.png';
+
+const demoMain = [{ label: 'Pull Request', value: 'pull-request' },
+  { label: 'Issues', value: 'issues' },
+  { label: 'Market', value: 'market' },
+  { label: 'Explore', value: 'explore' }];
+
+const demoOthers = [
+  { label: 'Overview', value: 'overview' },
+  { label: 'Repositories', value: 'Repositories' },
+  { label: 'Stars', value: 'Stars' },
+  { label: 'Followers', value: 'Followers' }];
+
+function Sidebar({ setMobileOpen }:{ setMobileOpen:Dispatch<SetStateAction<boolean>>}) {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.sideBar}>
+      <Link to="/" className={classes.imageLink}>
+        <img
+          src={githubWhite}
+          alt="GitHub Logo"
+          className={classes.image}
+        />
+      </Link>
+      <Divider />
+      <List>
+        <ListSubheader className={classes.subHeader}>MAIN</ListSubheader>
+        {demoMain.map(({ label, value }) => (
+          <Link key={value} className={classes.links} to="/">
```

Comment:
is this not meant to have `to={value}`?

Reply:
I try to use to={value} and see if it works the way I want it to.
The intention - is to open the content of the menus Area below the NavBar as shown in the design, hence not redirecting to a different path but home page, only changing the content inside the allocated Area.